mathesar/utils/tables.py

```diff
@@ -16,17 +22,37 @@ def get_table_column_types(table):
     return col_types
 
 
-def create_empty_table(data):
+def create_table_from_data(request, data):
+    name = data['name']
```

Comment:
I'm not sure that you need separate variables for these, you can just use them directly. It's not like they need to be calculated again if they're reused, you're just accessing a key in a dictionary.